### ISSUE TITLE
This needs to be moved to earlier in the JavaScript section

### ISSUE THREAD

**Issue by merecka** [learn-co-curriculum/fewpjs-js-fundamentals-comparisons#2]
This lab is very helpful, however, it needs to be moved to earlier in the JS section.  I had to look up a lot of this info in order to do previous JS labs before this one and since it is just now being presented in the curriculum it's not as helpful.  It would be much more helpful if it was delivered earlier in the section.

**Comment by sgharms** [comment]
Could you be more specific? Which labs required which knowledge?

We believed we covered "Just Enough" of these ideas in the lesson called: "introducing-the-dom-and-just-enough-javascript"

where we showed

![image](https://user-images.githubusercontent.com/50019/67503673-11564e80-f656-11e9-95d4-73047285b3c5.png)


What did we miss?

**Comment by merecka** [comment]
Well for one It doesn't talk about the difference between '==' and '===' which come to find out is pretty critical in JS.  And the timing of when this lab is presented in the curriculum just feels off.  While I think DOM manipulation is really fun & cool, I'm not sure why it is rushed into so fast in this JS curriculum.  It seems like spending 4-5 lessons going over the basics of JS syntax would probably build a better foundation before students get into DOM manipulation and actually start doing real coding.  This is similar to the way the Ruby curriculum is presented - start with the basics and move up.  Assuming that the students going through this JS curriculum have already done Ruby & Ruby on Rails, there doesn't need to be as much time spent on explaining basic concepts like "what is a loop", etc but I just think it would be better to just get the basic syntax out the way first rather than trying to skim over it with a very basic tutorial, then go into DOM manipulation (where complex functions & loops are needed), and then come back to the very basic JS syntax.    

Of course, this is just my personal opinion, but being a complete novice to JS when I started this curriculum I decided to offer my feedback (for whatever it's worth).
